schema_based_on_diagram.sql

```diff
@@ -41,4 +41,28 @@ CREATE TABLE Invoice_Items (
   treatment_id INT,
   FOREIGN KEY (invoice_id) REFERENCES Invoices(id),
   FOREIGN KEY (treatment_id) REFERENCES Treatments(id)
-);
\ No newline at end of file
+);
+
+-- Create join table for patient and treatments (many-to-many relationship)
+CREATE TABLE Patient_Treatments (
+  patient_id INT,
+  treatment_id INT,
+  PRIMARY KEY (patient_id, treatment_id),
+  FOREIGN KEY (patient_id) REFERENCES Patient(id),
+  FOREIGN KEY (treatment_id) REFERENCES Treatments(id)
+);
```

Comment:
![image](https://github.com/emhamza/database-schema-diagram/assets/93039359/055e6fd7-ca1e-4089-b348-49c8e9ffc2d3)

- [ ] Great work so far dear but it would be nice if you double-check the diagram because there is no many-to-many relationship between patients and treatments 😨 Kindly cross-check it and make the changes according to it 👍

Reply:
Hi @HAMMAS-SALEEM,

Thank you for your feedback. I overlooked it. `The Patient_Treatments` table has now been removed.